tests/bluetooth/controller/common/unit_sources.txt

```diff
@@ -0,0 +1,37 @@
+#
```

Comment:
we'll get rid of this file by creating a library for the files that make up the UUT (in a separate directory), so that our unittests has 3 libraries: 
- Unit Under Test
- common helper functions
- mock functions